packages/cxl-ui/src/components/cxl-adeft-star-rating.js

```diff
@@ -0,0 +1,52 @@
+import { customElement, html, LitElement, property } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@cwmr/iron-star-rating';
+
+@customElement('cxl-adeft-star-rating')
```

Comment:
Drop `adeft` here as well, I think general purpose `cxl-star-rating` will probably work.

Reply:
dropped in last commit